src/Examples/Akka.Hosting.CustomJournalIdDemo.FSharp/Program.fs

```diff
@@ -0,0 +1,318 @@
+                        let user = UserGenerator.CreateRandom()
+                        entityRegion.Tell(new CreateUser(user))
+                    )
+                Task.CompletedTask
+
+            override this.StopAsync(cancellationToken:CancellationToken) =
+                _cancelable.Cancel()
+                Task.CompletedTask
+
+open Akka.Actor
+open Akka.Cluster.Hosting
+open Akka.Cluster.Sharding
+open Akka.Hosting
+open CustomJournalIdDemo
+open Actors
+open Messages
+open Akka.Persistence.SqlServer.Hosting
+open Akka.Remote.Hosting
+
+module Main =
+    open Microsoft.AspNetCore.Builder
+    open Microsoft.Extensions.DependencyInjection
+    open System
+    open Microsoft.AspNetCore.Routing
+    open Microsoft.AspNetCore.Http
+    open System.Threading.Tasks
+    open System.Collections.Generic
+    open Microsoft.Extensions.Configuration
+    type RootHandler = delegate of ActorRegistry -> Task<IEnumerable<UserDescriptor>>
+    type UserHandler = delegate of string * ActorRegistry -> Task<UserDescriptor>
+
+    [<EntryPoint>]
+    let main argv =
+        let builder = WebApplication.CreateBuilder(argv)
+        
+        let akkaFun =
+            Action<_>(fun (configurationBuilder: AkkaConfigurationBuilder) ->   
+            // Grab connection strings from appsettings.json
+                let localConn = builder.Configuration.GetConnectionString("sqlServerLocal");
+                let shardingConn = builder.Configuration.GetConnectionString("sqlServerSharding");
+            // Custom journal options with the id "sharding"
+            // The absolute id will be "akka.persistence.journal.sharding"
+                let shardingJournalOptions = 
+                    new SqlServerJournalOptions(
+                        isDefaultPlugin = false
+                        , Identifier = "sharding"
+                        )
+                shardingJournalOptions.ConnectionString <- shardingConn
+                shardingJournalOptions.AutoInitialize <- false
+        
+            // Custom snapshots options with the id "sharding"
+            // The absolute id will be "akka.persistence.snapshot-store.sharding"
+                let shardingSnapshotOptions = 
+                    new SqlServerSnapshotOptions(
+                        isDefaultPlugin = false
+                        , Identifier = "sharding"
+                        )
+                shardingSnapshotOptions.ConnectionString <- shardingConn
+                shardingSnapshotOptions.AutoInitialize <- false
+                let co = new ClusterOptions()
+                co.Roles <- [| "myRole" |]
+                co.SeedNodes <- [| "akka.tcp://FAkkaHttp@localhost:8110" |]
+                let so = new ShardOptions ()
+                so.StateStoreMode <- StateStoreMode.Persistence
+                so.Role <- "myRole"
+                so.JournalPluginId <- shardingJournalOptions.PluginId
+                so.SnapshotPluginId <- shardingSnapshotOptions.PluginId
```

Comment:
I think its better to use `so.JournalOptions <- shardingJournalOptions` and `so.SnapshotOptions <- shardingSnapshotOptions` here

Reply:
Got it!! Since I found that there are still bugs... so I will close this first...Will create the PR  again later.

Reply:
Definitely please create a new PR!

Reply:
![image](https://user-images.githubusercontent.com/4289161/210685419-a506443c-6f9a-41d6-92e2-6f94642847f4.png)

Reply:
Might miss the plugin config...

Reply:
I'll look into this, thanks for the heads up!

Reply:
@ingted found the bug, it should be fixed with #186